code/app/src/main/java/com/adobe/marketing/edge/identity/app/model/SharedViewModel.kt

```diff
@@ -99,6 +119,29 @@ class SharedViewModel : ViewModel() {
         _authenticatedStateId.value = value
     }
 
+    fun getGAID(applicationContext: Context) {
+        viewModelScope.launch(Dispatchers.Default) {
+            Log.d("Shared_View_Model", "Thread working in ${Thread.currentThread().name}")
+            try {
+                val idInfo = AdvertisingIdClient.getAdvertisingIdInfo(applicationContext)
+                if (idInfo.isLimitAdTrackingEnabled) {
+                    Log.d("Shared_View_Model", "Limit Ad Tracking is enabled by the user, setting ad ID to \"\"")
```

Comment:
nit: For "SharedViewModel" you can use a single LOG_TAG constant for this class and reuse it everywhere you need it, this will help with renaming in the future.

Reply:
Thank you! That sounds good. I also had a couple questions regarding `LOG_TAG`:
- Is there a preferred scope for the `LOG_TAG` declaration in this case (I put it in the `companion object`)? 
- Do you know if there is a naming convention for the `LOG_TAG` string? That is, camel case, underscores, etc.
- Is it ok that the LOG_TAG is scoped to the view model and not the test app level? Should the specific view model/fragment be part of the log string instead?

Reply:
Thank you for all the information! This is super helpful; I've applied these changes to the logging system in the various views